Blockzilla/IntroViewController.swift

```diff
@@ -26,6 +26,104 @@ struct IntroViewControllerUX {
     static let FadeDuration = 0.25
 }
 
+protocol PageControlDelegate {
+    func selectedIndex(_ index:Int)
```

Comment:
2 nits, both of which possibly the code is not particularly compliant:

- delegate first arg should be unnamed param for the sender https://github.com/raywenderlich/swift-style-guide#delegates

- somewhat related: if not a delegate prefer`selected(index: Int)`, as we should start avoiding _ first args because that seems to be the latest Swift style. Feel free to correct me or ask for more info.